MapboxMobileEvents/MMEMetrics.h

```diff
@@ -9,6 +9,9 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic) NSUInteger totalDataTransfer;
 @property (nonatomic) NSUInteger cellDataTransfer;
 @property (nonatomic) NSUInteger wifiDataTransfer;
```

Comment:
Suggest making `readonly`, renaming to `wifiBytesTransfered` and computing the value